tests/integration/security/filebased_tls_origination/egress_gateway_origination_test.go

```diff
@@ -0,0 +1,470 @@
+//  Copyright Istio Authors
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package egressgatewayorigination
```

Comment:
nit: when you add the SDS based tls originiation test, we should have this in the same package? So probably shouldn't be named `filebased_tls_origination`? we can change it when we add that though

Also usually the package and folder name match, not a big deal though

Reply:
SDS secret tests will be in a different package, I'll change the package name here to `filebasedtlsorigination` when I add that test suite